python/hail/docs/getting_started_developing.rst

```diff
@@ -114,7 +114,16 @@ Several Hail tests have additional dependencies:
 
  - `QCTOOL 1.4 <http://www.well.ox.ac.uk/~gav/qctool>`_
 
- - `R 3.3.1 <http://www.r-project.org/>`_ with packages ``jsonlite``, ``SKAT`` and ``logistf``.
+ - `R 3.3.4 <http://www.r-project.org/>`_ with the normal R packages ``jsonlite``, ``SKAT`` and ``logistf``, as well as `pcrelate <https://www.rdocumentation.org/packages/GENESIS/versions/2.2.2/topics/pcrelate>`__ from the `GENESIS <https://bioconductor.org/packages/release/bioc/html/GENESIS.html>`__ *Bioconductor* package. This can be installed within R using:
+ 
+   .. code-block:: R
+
+      source("https://bioconductor.org/biocLite.R")
+      biocLite("GENESIS")
+      biocLite("SNPRelate")
+      biocLite("GWASTools")
+      
+   Newer verisons of R do not provide the ``ncdf`` dependency. If the above commands fail and mention ``ncdf`` or ``netcdf``, then you must install the netCDF package using your system's package manager. The name of the package is often ``ncdf`` or ``netcdf``.
```

Comment:
I vote we get rid of this line entirely. It sounds like this was a temporary failure on the part of the R people to package their product correctly. I think it adds more confusion than helps to our page, esp. since we state you should have R 3.3.4

Reply:
Ok. Actually, then remove `or higher`, it's redundant with the line below. And do change `This can be` to `The latter can be` (unless you intentionally avoided the latter...no pun).